### PR TITLE
Issue 49: Contributing.md included

### PR DESCRIPTION
**Pull request by Aparnarr**
Signed-off-by: Aparna <Aparna.Rr@emc.com>

**change log description**
1. The following folder and path is created
     documentation/src/docs/contributing.md
2. Contributing.md created and added
3. Links to wiki provided
4. contents for contribution guidelines added in the wiki 
      [wiki link]( https://github.com/pravega/pravega-operator/wiki/Contributing)

**Fixes issues**
  #49

### CODE REVIEW

File: documentation/src/docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Contributing to Pravega Operator
+
+Please check here for [Contributions guidelines](https://github.com/pravega/pravega-operator/wiki/Contributing)
+> Issue triaging and labeling
```
Comment (adrianmo) [inline documentation/src/docs/contributing.md]: Remove this line.

File: documentation/src/docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+> Review process
```
Comment (adrianmo) [inline documentation/src/docs/contributing.md]: Remove this line

File: documentation/src/docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Contributing to Pravega Operator
+
+Please check here for [Contributions guidelines](https://github.com/pravega/pravega-operator/wiki/Contributing)
```
Comment (adrianmo) [inline documentation/src/docs/contributing.md]: Replace with:

Please check the [Contributing](https://github.com/pravega/pravega-operator/wiki/Contributing) wiki page.

Comment (Aparnarr) [reply documentation/src/docs/contributing.md]: sure will edit

**Comment by Aparnarr** [comment]
@adrianmo 
All the comments were addressed and this PR is closed and replaced with PR #73.

**Pull request closed without merging**